src/targets/gpu/kernels/include/migraphx/kernels/hip.hpp

```diff
@@ -24,11 +24,17 @@
 #ifndef MIGRAPHX_GUARD_KERNELS_HIP_HPP
 #define MIGRAPHX_GUARD_KERNELS_HIP_HPP
 
+#ifndef MIGRAPHX_USE_HIPRTC
 // Workaround macro redefinition issue with clang tidy
 #if defined(__HIP_PLATFORM_HCC__) && defined(MIGRAPHX_USE_CLANG_TIDY)
 #undef __HIP_PLATFORM_HCC__ // NOLINT
 #endif
-
 #include <hip/hip_runtime.h>
+#include <hip/hip_fp16.h>
+#include <hip/math_functions.h>
+#else
+#include <hip/hip_common.h>
+#endif
+#include <hip/hip_math_constants.h>
```

Comment:
This include needs a `#ifdef MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS` as well.

Reply:
Done.